helpers/dbload.ts

```diff
@@ -19,7 +19,7 @@ const options: Options = {
   }
 }
 
-if (process.env.NODE_ENV != 'production') {
+if (process.env.NODE_ENV === 'development') {
```

Comment:
We need more clarify on why this check is added. It should be always used.

Reply:
I removed this check altogether. Not dbload uses `DB_PORT` env variable and next config has fallback in case if this variable was not configured. It looks much cleaner IMO and it should work in production (assuming postgres is running on default 5432 port).